test/BUILD

```diff
@@ -432,3 +435,8 @@ scala_binary(
     main_class = "scala.test.BinaryDependentOnJava",
     deps = [":JavaOnlySources"],
 )
+
+scala_proto_library(
+    name = "test_proto",
+    deps = ["//test/proto:test2", "//test/proto:test3"]
+)
```

Comment:
can you add a test that creates these things in some scala code?

Reply:
Sure, let me do that... I'll also add a mock grpc service to see how that works too